android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebView.java

```diff
@@ -15,6 +15,7 @@
 public class MagnetWebView extends WebView implements LifecycleEventListener {
     String TAG = "MagnetWebView";
```

Comment:
Let's take the opportunity to make this `private static final String TAG = "MagnetWebView"`.